gluten-core/src/main/scala/org/apache/spark/memory/SparkMemoryUtil.scala

```diff
@@ -58,6 +58,7 @@ object SparkMemoryUtil {
           overAcquire.getTarget.accept(this)
         }
 
+        @scala.annotation.nowarn
```

Comment:
Is the "deprecated RegularMemoryConsumer" warning meant to be ignored here?

Reply:
Yes. The annotation is required otherwise compilation will fail.